test/cypress/integration/group-contributions.spec.js

```diff
@@ -371,7 +371,8 @@ describe('Contributions', () => {
 
   it('user1 edits the non monetary contribution', () => {
     cy.getByDT('buttonEditNonMonetaryContribution').click()
-    cy.getByDT('inputNonMonetaryContribution').type('{selectall}{del}French classes{enter}')
+    cy.getByDT('inputNonMonetaryContribution').clear()
+    cy.getByDT('inputNonMonetaryContribution').type('French classes{enter}')
```

Comment:
This is a fix for a heisen-bug I've seen a few times before (e.g. [here](https://cloud.cypress.io/projects/q6whky/runs/3578/test-results/28ff40ce-17e5-42b6-8010-430d614543ab?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%5D&statuses=%5B%7B%22value%22%3A%22FAILED%22%2C%22label%22%3A%22FAILED%22%7D%5D&testingTypesEnum=%5B%5D&utm_source=github)). `type({selectall}{del})` is converted to `clear()` here. Also, Cypress doc warns that it's unsafe to chain something onto `clear()`. So breaking it into two lines here.